src/sentry/static/sentry/app/components/activity/item/index.tsx

```diff
@@ -89,8 +107,26 @@ function ActivityItem({
           <ActivityHeader>
             <ActivityHeaderContent>{header}</ActivityHeaderContent>
 
-            {date && showDate && !showTime && <StyledTimeSince date={date} />}
-            {date && showDate && showTime && <StyledDateTime timeOnly date={date} />}
+            {date &&
+              showdate &&
+              !showDateEnded &&
+              (showTime ? (
+                <StyledDateTime timeOnly date={date} />
+              ) : (
+                <StyledTimeSince date={date} />
+              ))}
```

Comment:
This is kind of a confusing ternary... I would refactor to make an intermediate variable for `date && showDate && !showDateEnded` so that we have more context of what this set of conditions means -- I can see this being misinterpreted with the following ternary.